pkg/k8s/pods/pod.go

```diff
@@ -178,5 +178,47 @@ func Restart(dev *model.Dev, c *kubernetes.Clientset) error {
 		}
 	}
 
-	return nil
+	if !wait {
+		return nil
+	}
+
+	return waitUntilRunning(dev.Namespace, fmt.Sprintf("%s=%s", OktetoDetachedDevLabel, dev.Name), c)
+}
+
+func waitUntilRunning(namespace, selector string, c *kubernetes.Clientset) error {
+	t := time.NewTicker(1 * time.Second)
```

Comment:
I would add logs telling we are waiting for pods to be ready and a final log telling everything is up and running. Maybe a log for each pod that reaches the running state too.

Reply:
I'm not a big fan of showing too many logs. I like linux'  no news is good news approach. How about we just show a log saying 'waiting for pods to restart', and the success one? @rlamana what do you think?

Reply:
sounds good

Reply:
I also think that way. "Waiting > Success" sounds good to me.